openedx/core/djangoapps/course_live/tests/test_views.py

```diff
@@ -266,3 +268,59 @@ def test_response_has_correct_data(self):
         response = self.client.get(self.url)
         content = json.loads(response.content.decode('utf-8'))
         self.assertEqual(content, expected_data)
+
+
+class TestCourseLiveIFrameView(ModuleStoreTestCase, APITestCase):
```

Comment:
I think a few more test cases are needed.
1. Test with a user that is not enrolled in the course.
2. Test when live configuration exists but is not enabled.
3. Test live configuration does not exist for the course.